### DISCUSSION TITLE
Model generation files missing

### DISCUSSION THREAD

**Discussion by LucianKennelly** [simnibs/simnibs#171] [2. General]
Hello all,
I have just recently been using NEURON and SimNIBS to model the affect of a TMS device on the brain, yet I am having a bizzare issue with generating a model in matlab, since I am able to successfully generate a model on an older computer, yet on my current computer (which I have tried to set up exactly as the old one) the model generation fails but NEURON launches and all other NEURON functionality is fine, yet there are a few files of code missing from the NEURON folder in the model generation. Could anyone help me?

**Comment by TorgeW** [comment]
Hey,

The main question is, what code are you running? There is currently no connection between simNIBS and NEURON implemented in this github. And another question would be what exactly is missing. If you provide me with this information i might be able to help..

**Reply by LucianKennelly** [reply]
The thing is, it is from this github that I download the program that is bugged. It happens when I run the matlab model generation code, which then generates a model folder with missing files in the Code/NEURON folder such as init_pp.hoc, init_rec.hoc, among other .hoc files and it is missing all the .dat files as well.

**Reply by TorgeW** [reply]
This is the SimNIBS github, we dont have any matlab code to generate models that are related to NEURON in the current version.
